alarm/utils.py

```diff
@@ -33,3 +33,55 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def create_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in thresholds:
```

Comment:
So it does not care if threshold was broken or not.

Reply:
I think this is important because they should only create an object in the database in the ThresholdBreak model if the price breaks the threshold